src/sphinx_codelinks/cmd.py

```diff
@@ -268,7 +269,28 @@ def write_rst(  # noqa: PLR0913  # for CLI, so it takes as many as it requires
 ) -> None:
     """Generate needextend.rst from the extracted obj in JSON."""
     logger.configure(verbose, quiet)
-    convert_marked_content(jsonpath, outpath, remote_url_field, title)
+    try:
+        with jsonpath.open("r") as f:
+            marked_content = json.load(f)
+    except Exception as e:
```

Comment:
have you head of https://docs.astral.sh/ruff/rules/blind-except/#blind-except-ble001 ?